app/controllers/story_urls_controller.rb

```diff
@@ -9,6 +9,10 @@ def all
         render json: Story.find_similar_by_url(url).for_presentation
       }
     end
+  rescue ActionController::ParameterMissing
+    respond_to do |format|
+      format.json { render json: {}, status: :bad_request }
+    end
   end
 
   def latest
```

Comment:
same exception is raised for this action too. What should be the ideal way to handle it ?

Reply:
I spent a minute looking into it on stream today, a viewer pointed out that this should probably happen more universally by catching the `ParameterMissing` exception in middleware. The Strong Parameters docs do recommend using the raise/rescue pattern, and I realized we already do this for the other exception it raises.

If you look at ApplicationController, line 34 rescues `UnpermittedParameters`. Could you move your rescue up alongside that?

Ideally, your `respond_to` block could also return a `plain` version, and the existing rescue for `UnpermittedParameters` could do that, too. But I realize that's extra work and I can do it if you don't want to.

Reply:
I have made changes as per above suggestion and updated few specs accordingly. Please do check.